Documentation/contributing.rst

```diff
@@ -890,19 +890,21 @@ regarding which phrase triggers which build, which build is required for
 a pull-request to be merged, etc. Each linked job contains a description
 illustrating which subset of tests the job runs.
 
-+-------------------------------------------------------------------------------------------------------+-----------------+--------------------+
-| Jenkins Job                                                                                           | Trigger Phrase  | Required To Merge? |
-+=======================================================================================================+=================+====================+
-| `Cilium-Master-Bash-Tests-All <https://jenkins.cilium.io/job/Cilium-PR-Bash-Tests-All/>`_             | test-me-please  | Yes                |
-+-------------------------------------------------------------------------------------+-----------------+-----------------+--------------------+
-| `Cilium-PR-Ginkgo-Tests-Validated <https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/>`_ | test-me-please  | Yes                |
-+-------------------------------------------------------------------------------------+-----------------+-----------------+--------------------+
-| `Cilium-PR-Ginkgo-Tests-All <https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-All/>`_             | test-all-ginkgo | No                 |
-+-------------------------------------------------------------------------------------------------------+-----------------+--------------------+
-| `Cilium-Pr-Ginkgo-Test-k8s <https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-k8s/>`_              | test-missed-k8s | No                 |
-+-------------------------------------------------------------------------------------------------------+-----------------+--------------------+
-| `Cilium-Nightly-Tests-PR <https://jenkins.cilium.io/job/Cilium-PR-Nightly-Tests-All/>`_               | test-nightly    | No                 |
-+-------------------------------------------------------------------------------------------------------+-----------------+--------------------+
++----------------------------------------------------------------------------------------------------------+-------------------+--------------------+
+| Jenkins Job                                                                                              | Trigger Phrase    | Required To Merge? |
++==========================================================================================================+===================+====================+
+| `Cilium-Master-Bash-Tests-All <https://jenkins.cilium.io/job/Cilium-PR-Bash-Tests-All/>`_                | test-me-please    | Yes                |
```

Comment:
I know this is related to this PR specifically, but can you remove `Cilium-Master-Bash-Tests-All` from this? it is no longer triggered with `test-me-please`.